### PR TITLE
Cell bits reordered; Geometry fixed; LG/HG ratio fixes

### PR DESCRIPTION
**Pull request by peressounko**

### CODE REVIEW

File: Detectors/PHOS/simulation/src/Detector.cxx [hunk 1]
```diff
@@ -972,16 +972,16 @@ void Detector::addAlignableVolumes() const
...
-      matTtoL->MultiplyLeft(&globmatrixi);
-      alignableEntry->SetMatrix(matTtoL);
+      // Float_t angle = geom->getPHOSAngle(iModule);
+      // TGeoHMatrix* globMatrix = alignableEntry->GetGlobalOrig();
+
+      // TGeoHMatrix* matTtoL = new TGeoHMatrix;
+      // matTtoL->RotateZ(270. + angle);
+      // const TGeoHMatrix& globmatrixi = globMatrix->Inverse();
+      // matTtoL->MultiplyLeft(&globmatrixi);
+      // alignableEntry->SetMatrix(matTtoL);
```
Comment (shahor02) [inline Detectors/PHOS/simulation/src/Detector.cxx]: if this is code is needed, please remove it.

Comment (peressounko) [reply Detectors/PHOS/simulation/src/Detector.cxx]: Removed, but (squashed) update does not appear in this PR. What could be the reason? Should I close this PR and open another?

Comment (shahor02) [reply Detectors/PHOS/simulation/src/Detector.cxx]: @peressounko I think you should rebase your branch to fresh dev then force-push

**Comment by peressounko** [comment]
So far I did not do rebase till merging of PR. This time I rebased between commits. Most of modified 93 files are from rebase

**Comment by shahor02** [comment]
Rebase cannot have such effect, I suspect you did merge. You should probably rename this branch to something else,
`git branch -b xxx`,  then reset your dev to clean origin/dev, `git checkout dev; git reset --hard origin/dev`, update it: `git pull`, create a new branch `git checkout -b pr_bits`, then cherry pick your commit `git cherry-pick 2ca17e2eb0844f989ddaae31de01ec7127e34d5c`. Then just open a new PR.
Actually, opening a PR from dev is not a good practice.

**Pull request closed without merging**